src/main/webapp/assets/modules/common/challenge/submissionChallenge.js

```diff
@@ -88,7 +91,7 @@ techlooper.directive("submissionChallenge", function (localStorageService, apiSe
                   }, 500);
                 });
             }
-            scope.submissionForm.submissionURL.$setValidity("invalidUrl", !inValid);
+            scope.submissionForm && scope.submissionForm.submissionURL.$setValidity("invalidUrl", !inValid);
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.